Mathlib/Analysis/SpecialFunctions/ContinuousFunctionalCalculus/Rpow/Basic.lean

```diff
@@ -319,6 +404,51 @@ lemma rpow_intCast (a : Aˣ) (n : ℤ) (ha : (0 : A) ≤ a := by cfc_tac) :
   refine cfc_congr fun _ _ => ?_
   simp
 
+section prod
+
+variable [IsTopologicalRing A] [T2Space A]
+variable {B : Type*} [PartialOrder B] [Ring B] [StarRing B] [TopologicalSpace B]
+  [StarOrderedRing B]
+  [Algebra ℝ B] [ContinuousFunctionalCalculus ℝ B IsSelfAdjoint]
+  [ContinuousFunctionalCalculus ℝ (A × B) IsSelfAdjoint]
+  [IsTopologicalRing B] [T2Space B] [StarOrderedRing (A × B)]
+  [NonnegSpectrumClass ℝ B] [NonnegSpectrumClass ℝ (A × B)]
+
+/- Note that there is higher-priority instance of `Pow (A × B) ℝ` coming from the `Pow` instance for
+products, hence the direct use of `rpow` here. -/
+lemma rpow_map_prod {a : A} {b : B} {x : ℝ} (ha : 0 ∉ spectrum ℝ≥0 a) (hb : 0 ∉ spectrum ℝ≥0 b)
+    (ha' : 0 ≤ a := by cfc_tac) (hb' : 0 ≤ b := by cfc_tac) :
+    rpow (a, b) x = (a ^ x, b ^ x) := by
```

Comment:
This comment makes me wonder:

1. Should we add this to the module docstring as an example of why we have the lower priority in the first place?
2. Should this be phrased as `rpow ab x = ab ^ x` for `ab : A × B`? You could call that `rpow_eq_rpow_prod`? You could keep this lemma around too for convenience.

Reply:
Done; we now have the two versions of the four lemmas.